client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/hooks.tsx

```diff
@@ -42,7 +46,26 @@ export const useDraftRequisitionLine = (item?: ItemRowFragment | null) => {
     }
   };
 
-  return { draft, isLoading, save: () => draft && save(draft), update };
+  const save = async () => {
+    if (draft) {
+      let result = await saveAction(draft);
+      if (
+        result.updateResponseRequisitionLine.__typename ===
+        'UpdateResponseRequisitionLineError'
+      ) {
+        switch (result.updateResponseRequisitionLine.error.__typename) {
+          case 'ReasonNotProvided':
+            error(t('error.requires-reason'))();
+            break;
+          default:
+            error(result.updateResponseRequisitionLine.error.description)();
+            break;
```

Comment:
Handling error toast inside hook.

I see there are a number of ways that we are handling errors - in the api, in the hooks, or in components. 

Would love to add some consistency to our patterns!